src/modules/common/components/RemoteContent.js

```diff
@@ -3,6 +3,7 @@
 import React from 'react'
 
 import styled, { css } from 'styled-components'
+import { push } from 'redux-first-router'
```

Comment:
Don't use a router-dependency in RemoteContent since we have it in the shared-lib

Reply:
Done, moved back to pages.